tests/load/locustfiles/locustfile.py

```diff
@@ -123,6 +139,31 @@ def hello(self) -> None:
             timer.response_length = len(reply.encode("utf-8"))
         self.uaid = res.uaid
 
+    def ack(self) -> None:
+        """
+        Send an 'ack' message to push.
+
+        After sending a notification, the client must also send an 'ack' to the server to
+        confirm receipt. If there is a pending notification, this will try and receive it
+        before sending an acknowledgement.
+        """
+        with self._time_event(name="acknowledge"):
+            channel_id = list(self.channels.keys())[-1]
+            body = json.dumps(
+                dict(messageType="ack", use_webpush=True, updates=dict(channelID=channel_id))
+            )
+
+            self.connect()
+            self.hello()
```

Comment:
In the case where we are re-connecting. I think we want to re-use the `uaid`? I think we need to add support in the hello() method for this.

Reply:
Check if user already has UAID and reuse it